src/tenable/load_vulnerabilities.py

```diff
@@ -93,10 +93,10 @@
 """
 
 
-def load_tenable_vulnerabilities(export: VulnExportStatus, to: mariadb.Connection):
```

Comment:
Pass a `ConnectionPool` here instead of removing the parameter altogether - this avoids having to monkey patch CONN_PARAMS, which is generally a bad idea.